### ISSUE TITLE
The text in the FilesMaster Flatpak title bar is only displayed when hovered over

### ISSUE THREAD

**Issue by Ads20000** [ubuntu/yaru#914]
Thank you for contributing to **Yaru**, the next Ubuntu default Theme made by the community.

If you found a bug please consider to fill below information, this will help us to understand the problem and we don't bother you with other questions :)

If you're filing an issue for a missing icon, look for **Icon Request** section below.

Thanks a lot!

------ ISSUE TEMPLATE starts HERE ------
## Expected Behavior

Use the FilesMaster 3.30.0-7de42f4 Flatpak with Yaru with no theming problems.

## Actual Behavior

The text in the title bar is only displayed when hovered over.

![filesmaster icons](https://user-images.githubusercontent.com/3178485/47022599-c574ed80-d155-11e8-88f6-655e51f6ae14.gif)

## Steps to Reproduce the Problem

1. `sudo apt install flatpak`
2. `flatpak remote-add --from gnome-apps-nightly https://sdk.gnome.org/gnome-apps-nightly.flatpakrepo`
3. `flatpak install gnome-apps-nightly org.gnome.NautilusMaster`
4. `flatpak run org.gnome.NautilusMaster`
5. Hover over the active folder part of the title bar

## Specifications

```
$ snap info communitheme
name:      communitheme
summary:   The next Ubuntu theme built by the community.
publisher: Didier Roche (didrocks)
license:   unset
description: |
  Yaru, formerly known as Communitheme, is the new Ubuntu theme built by the
  community. Yaru will become the default Ubuntu theme in Ubuntu 18.10. This
  package allows you to try out the theme on Ubuntu 18.04 LTS.
  
  To try out the theme, install this package on Ubuntu 18.04 LTS, restart
  your computer and select the "Ubuntu with communitheme snap" session from
  the login screen.
  
  More information is available at
  https://community.ubuntu.com/t/faq-ubuntu-new-theme/1930.
snap-id:      Yd6CISPIf6tEf3ZEJ0cqSoEg9rG2VkRi
tracking:     edge
refresh-date: 6 days ago, at 18:54 BST
channels:                    
  stable:    0.1 (1273) 17MB -
  candidate: ↑               
  beta:      ↑               
  edge:      0.1 (1275) 17MB -
installed:   0.1 (1275) 17MB -

$ snap version
snap    2.35.5
snapd   2.35.5
series  16
ubuntu  18.04
kernel  4.15.0-36-generic

$ snap info core
tracking:     beta
refresh-date: today at 09:42 BST
installed:   16-2.35.5                   (5742) 92MB core
```

Software that presents the issue:
- Name: FilesMaster
- Version: 3.30.0-7de42f4

**Comment by godlyranchdressing** [comment]
That'd be the new pathbar design, which I don't think we ever worked on. I don't know if we should fix it or come up with a new mockup, though. 

@clobrano @madsrh do you think we should mimic an entry field or stick to the old underline design? I haven't seen it live, so I have no idea if it changed but it does look very much like a regular entry field in Adwaita.

![nautilus-3-30](https://user-images.githubusercontent.com/27529229/47024096-0a395e00-d12f-11e8-99d2-137128ebb39e.jpg)

**Comment by clobrano** [comment]
Correct, we cannot style the same widget it two different ways. Until Ubuntu won't ship latest Nautilus we must keep the current styling.

@godlyranchdressing, I agree with you, I expect the entry style to be the fittest for this

**Comment by clobrano** [comment]
very nice indeed!

> @clobrano in that case, should we tag this issue as "future release" or something?

yes, this should do it

**Comment by sagcan** [comment]
> in that case, should we tag this issue as "future release" or something?

@godlyranchdressing have you merged your fix with upstream? If not, does this mean that non Ubuntu users have to wait for Ubuntu 19.04 to have a functional Yaru theme (apart from fixing this locally)?

**Comment by clobrano** [comment]
@sagcan, in this case we are upstream. Flatpak get source code from a specific commit, right now, when we will have a branch for 19.04 we can change that reference, so that flatpak will be updated as well

**Comment by Feichtmeier** [comment]
Just to make sure we considered this idea:

For now, since we still don't know if 19.4 will update to upstream nautilus, we could adapt the stackswitcher styling to the buttonbox styling we use in gnome software

This would need to be tested with the new nautilus ofc 

Opinions? @godlyranchdressing @clobrano @madsrh

**Comment by clobrano** [comment]
> we could adapt the stackswitcher styling to the buttonbox styling we use in gnome software

I am not sure to understand the goal. To make current path bar looks like the new one?

**Comment by Feichtmeier** [comment]
No, to provide a theming without the "broken" underline :)

**Comment by clobrano** [comment]
I could be a good idea

**Comment by Feichtmeier** [comment]
Would look like this

![image](https://user-images.githubusercontent.com/15329494/48199227-cf0c0280-e35b-11e8-915d-1870f398e189.png)

![image](https://user-images.githubusercontent.com/15329494/48199260-e5b25980-e35b-11e8-8d80-ea36bc6af0c2.png)

**Comment by clobrano** [comment]
> Edit2: ofc we would need to test this for the flatpak. If it doesnt work anyways it's basically useless, apart from unifying the looks, which is not a high prio imho

I agree, if you can made a PR, I should be able to test Nautilus master later today

**Comment by Feichtmeier** [comment]
Yaku from the hub has mentioned that the staging PPA has nautilus 3.30 for cosmic I'll check it out this weekend (if you are not faster @clobrano ;) )

**Comment by Feichtmeier** [comment]
PPA works! But you need to install tracker after adding the ppa or nautilus won't start =)

Anyways.... this looks bad, we need another idea 
![image](https://user-images.githubusercontent.com/15329494/48266422-9dfd0200-e42f-11e8-9151-37fecb356a76.png)

I wonder how @godlyranchdressing  's styling would look on nautilus 3.26 ?
